package.json

```diff
@@ -56,7 +56,19 @@
     "jest-test": "jest __test__/backend.test.js",
     "dev": "nf start -w --procfile ./dev-tools/Procfile.dev",
     "debug-server": "nf start -w --procfile ./dev-tools/Procfile-debug-server.dev",
-    "buy-numbers": "./dev-tools/babel-run-with-env.js ./dev-tools/buy-numbers.js"
+    "buy-numbers": "./dev-tools/babel-run-with-env.js ./dev-tools/buy-numbers.js",
+    "kathy-test-components": "yarn test __test__/components",
```

Comment:
We don't want to commit `kathy-` package.json entries to main (unless there's a really particular reason why) -- if you find these utilities helpful, we could rename them without your name on them? Or maybe these are just nice utilities that you like and we don't want them on main?

Reply:
Sorry I was using those to debug some stuff and totally forgot to remove them! Yes, definitely shouldn't be on main. I've removed them